src/components/CompaniesGrid/index.tsx

```diff
@@ -0,0 +1,62 @@
+import s from "./style.module.scss";
+
+const companies = [
+  {
+    name: "Shield AI",
+    logo: "",
```

Comment:
Let's comment out the companies with missing logos for now so we can get it merged and add them later once we have the correct image

![Screenshot 2022-11-11 at 2 07 07 AM](https://user-images.githubusercontent.com/33165426/201317353-a1263ddb-2519-436f-beb9-63e21e7ecfd6.jpg)